Assets/MixedRealityToolkit/_Core/Definitions/InteractionDefinition.cs

```diff
@@ -9,6 +13,9 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Definitions
     /// </summary>
     public struct InteractionDefinition
     {
+
+        #region Interaction Properties
+
         /// <summary>
         /// The ID assigned to the Input
```

Comment:
Not sure that I agree with the ID being a string. Let's follow the pattern of the rest of the ids we keep as uints

Reply:
Agreed, I think it was an oversight